byzcoinx/byzcoinx_test.go

```diff
@@ -118,7 +118,8 @@ func TestBftCoSi(t *testing.T) {
 	require.Nil(t, err)
 
 	for _, n := range []int{1, 2, 4, 9, 20} {
-		runProtocol(t, n, 0, 0, protoName)
+		nbrFault := FaultThreshold(n)
+		runProtocol(t, n, nbrFault, 0, protoName)
```

Comment:
shouldn't this test check the scenario where we do not have any faults? because the faulty test is covered by `TestBftCoSiFault`